packages/agent/src/environment.ts

```diff
@@ -31,7 +32,7 @@ const toBoolean = (value: string | undefined, defaultValue?: boolean): boolean =
 export const ENV_VAR_PREFIX = process.env.ENV_VAR_PREFIX ?? ''
 export const DB_TYPE = env('DB_TYPE', ENV_VAR_PREFIX) ?? 'postgres'
 //#DB_URL="database/agent_default.sqlite"
-export const DB_URL = env('DB_URL', ENV_VAR_PREFIX) ?? 'postgresql://postgres:your-super-secret-and-long-postgres-password@127.0.0.1:5432/postgres'
+export const DB_URL = env('DB_URL', ENV_VAR_PREFIX) // Using DB_URL is optional
```

Comment:
At least then make a comment what the URL can look like

Reply:
fixed